.gitignore

```diff
@@ -109,3 +109,7 @@ libzcashconsensus.pc
 
 contrib/debian/files
 contrib/debian/substvars
+
+src/fuzzing/*/output
+src/fuzz.cpp
+src/fuzz.h
```

Comment:
It would be good to add these two copied files to `make clean`, so they aren't around during normal development.